android/sdk/src/main/java/org/jitsi/meet/sdk/ConnectionService.java

```diff
@@ -395,9 +389,7 @@ public void onHold() {
             // What ?! Android will still call this method even if we do not add
             // the HOLD capability, so do the same thing as on abort.
             // TODO implement HOLD
-            Log.d(TAG, String.format(
-                  "onHold %s - HOLD is not supported, aborting the call...",
-                  getCallUUID()));
+            JitsiMeetLogger.w(TAG + "onHold %s - HOLD is not supported, aborting the call... " + getCallUUID());
```

Comment:
The logger itself formats too, I went with the smallest amount of change, but I can adjust.

Reply:
What I mean is that the logger won't get a chance to format this one, because it's passed as a string(`+` operator is used).

Reply:
There is nothing special to format, is there? I mean, we log the same way elsewhere in the file too. Or am I missing something?

Reply:
`JitsiMeetLogger.w(TAG + " onHold %s - HOLD is not supported, aborting the call... " + getCallUUID());`

should be:

`JitsiMeetLogger.w(TAG + " onHold %s - HOLD is not supported, aborting the call... ", getCallUUID());`

Notice how `,` is used instead of `+` before the `getCallUUID()`.

I mean to pass `getCallUUID()` as an argument instead of concatenating it with the log string. The logger will not format without any arguments.

Reply:
aaaaah, that! 🤦‍♂  sorry I misread!